doc/tests/README.md

```diff
@@ -70,15 +80,85 @@ In the application the integration tests will be used to test the following area
 In the backend integration tests can be found in the `src/test/` directory.
 The whole module is annotated with `#[cfg(test)]` and will therefore only be compiled for tests.
 
-## Manual Testing
+### 3. System Testing
 
-Manual testing will include performing an end-to-end testing of the application to ensure overall functionality and user experience.
+System testing will include performing an end-to-end testing of the application to ensure overall functionality and user experience.
```

Comment:
```suggestion
System testing will include performing and end-to-end testing of the application to ensure overall functionality and user experience.
```

Reply:
it was jsut bad english from my side, I rewrote it to: "System testing will include end-to-end testing of the application to ensure overall functionality and user experience."